Cargo.toml

```diff
@@ -74,7 +75,7 @@ std         = [] # support for no_std in a backwards-compatible way
 suggestions = ["strsim"]
 color       = ["ansi_term", "atty"]
 wrap_help   = ["term_size", "textwrap/term_size"]
-derive      = ["clap_derive"]
+derive      = ["clap_derive", "lazy_static"]
```

Comment:
We can move the dependency to the derive crate

Reply:
Nope. We need to reexport lazy_static, but proc-macro crates cannot export anything but proc-macros.